chromium_src/chrome/common/media/cdm_host_file_path.cc

```diff
@@ -0,0 +1,8 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#define GOOGLE_CHROME_BUILD
```

Comment:
this is very dangerous, it could leak `GOOGLE_CHROME_BUILD` into any of the header files included by cdm_host_file_path.cc. It's also not sufficient to include them before this define because they could change

Reply:
it looks like we are doing this in some other places too so at the very least we should add all the includes from `cdm_host_file_path.cc` so we don't accidentally set it in other files

Reply:
I thought we could detect the future header change by this because this will only affect header.
Ok, including headers in advance would be much safer.